docs/developer-docs/latest/plugins/email.md

```diff
@@ -121,4 +121,4 @@ module.exports = {
         }
     }
 }
-```
\ No newline at end of file
+```
```

Comment:
Not sure why git insists to include this in that PR 😅  You can safely ignore it 🤷

Reply:
It might be a mandatory line break at the end of the file that was added by a parser